dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -426,15 +510,37 @@ def __init__(self, pipeline, **kwargs):
                 dataset_impl = _DALIDatasetImpl(pipeline, **kwargs)
                 super(DALIDatasetWithInputs, self).__init__(dataset_impl, dataset_options())
 
-
-        current_module = sys.modules[__name__]
-        experimental_module = _internal.get_submodule(current_module, "experimental")
-        DALIDatasetWithInputs.__module__ = experimental_module
         DALIDatasetWithInputs.__doc__ = _experimental_dataset_docstring
-        setattr(experimental_module, "DALIDatasetWithInputs", DALIDatasetWithInputs)
+        _insert_experimental_member(DALIDatasetWithInputs, "DALIDatasetWithInputs")
+
+        class input:
```

Comment:
why not `Input`? I personally got confused when I saw `experimental.input` earlier, as I thought it was an instance

Reply:
Done